package.json

```diff
@@ -33,8 +33,7 @@
   },
   "commitlint": {
     "extends": [
-      "@commitlint/config-conventional",
-      "@commitlint/config-lerna-scopes"
+      "@commitlint/config-conventional"
```

Comment:
If we are doing the config via `commitlint.config,js`, lets move everything there and remove the config from `package.json`